server/src/service/BoardService.js

```diff
@@ -133,4 +132,14 @@ export class BoardService extends BaseService {
         const createInvitation = this.invitationRepository.create(invitation);
         await this.invitationRepository.save(createInvitation);
     }
+
+    @Transactional()
+    async updateBoard(hostId, boardId, title) {
+        const board = await this.customBoardRepository.findBoardById(boardId);
+        if (!board) throw new EntityNotFoundError();
+        this.checkForbidden(hostId, boardId);
+        const boardToUpdate = await this.boardRepository.findOne(boardId);
+        boardToUpdate.title = title;
+        await this.boardRepository.save(boardToUpdate);
```

Comment:
```suggestion
        const board = await this.customBoardRepository.findBoardById(boardId);
        if (!board) throw new EntityNotFoundError();
        this.checkForbidden(hostId, boardId);
        board.title = title;
        await this.boardRepository.save(board);
```
138번째 줄의 코드와 141번째 줄의 코드가 어떤 차이가 있나요?
두 번 조회를 하는 이유가 궁금해요🧐

Reply:
커스텀으로 분리하면서 착오가 있었나봅니다 ㅜ
 리뷰 반영하였습니다 ~